### PR TITLE
added support for carthage checkout command

### PR DESCRIPTION
**Pull request by KoCMoHaBTa**

### CODE REVIEW

**Comment by viktorbenei** [comment]
@vasarhelyia looks good to me, wdyt?

File: step.swift [hunk 1]
```diff
@@ -108,7 +109,13 @@ if bootstrapCommand && hasCachedItems {
 }
 
 let command = "carthage \(carthageCommand)"
-var args = " " + ( collectArgs(env).map { "\($0)" } ).joinWithSeparator(" ")
+var args = " "
+
+if !checkoutCommand {
+
```
Comment (viktorbenei) [inline step.swift]: unnecessary empty line

**Comment by viktorbenei** [comment]
@vasarhelyia LGTM

**Comment by vasarhelyia** [comment]
awesome, thanks! GTM

**Pull request merged**